Common/UnitDefinitions/ElectricPotentialAc.json

```diff
@@ -1,20 +0,0 @@
-{
-  "Name": "ElectricPotentialAc",
```

Comment:
Just a heads up that we can't delete these, we have to mark them obsolete in JSON with `ObsoleteText` on the quantity for example just below `Name` on line 3, then add to #1200 , then remove them in next major version.

The obsolete text should describe what they should use instead for AC and DC operations.
This can wait until the design is more complete of course.